assignment8/strings2.js

```diff
@@ -0,0 +1,10 @@
+exports.lengthOfLastWord = function(a){
```

Comment:
This function doesn't return the length of the last word, it returns the length of the first word 😆